src/main/kotlin/day15/Repository/Users.Repository.kt

```
@@ -0,0 +1,29 @@
+    }
+
+    override suspend fun getUsersTop10ByUserName(): List<User> {
+        return users.sortedBy { user -> -user.id }.slice(0..9)
+    }
+}
+
+
+const val userDataSource = "./src/main/kotlin/day15/RawData/users.json"
+
+class UserRepository : UserRepositoryImpl, UserDataSourceImpl(readJson(userDataSource)) {
+}
```

detekt (1.23.7)

[warning] The class or object UserRepository is empty.

(detekt.empty-blocks.EmptyClassBlock)